packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -559,7 +551,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
             voter,
             msg.sender,
             currentRoundId,
-            _getPriceRequest(identifier, time, ancillaryData).pendingRequestIndex,
+            _getPriceRequest(identifier, time, ancillaryData).priceRequestIndex,
```

Comment:
this was a bug that I found!!! when we emit `VoteRevealed` we need to include the `priceRequestIndex` NOT the `pendingRequestIndex`